src/commands/org/open.ts

```diff
@@ -130,48 +123,21 @@ export class OrgOpenCommand extends SfCommand<OrgOpenOutput> {
       throw err;
     }
 
-    const openOptions = flags.browser
-      ? // assertion can be removed once oclif option flag typings are fixed
-        { app: { name: apps[flags.browser as 'chrome' | 'edge' | 'firefox'] } }
-      : {};
+    const openOptions =
+      // assertions could be removed once oclif flag typins are fixed
+      flags.browser ? { app: { name: apps[flags.browser as 'chrome' | 'edge' | 'firefox'] } } : {};
```

Comment:
_I think_ we can remove the assertion if we make use the new `Flags.option` for `flags.browser`.

Reply:
I closed 480.  I'll move this flag change idea to https://github.com/salesforcecli/plugin-org/pull/917